cms/startup.py

```diff
@@ -51,6 +51,12 @@ def run():
     # validate configurations on startup
     validate_cms_config(settings)
 
+    import gc
+    import logging
+    log = logging.getLogger(__name__)
+    log.info(gc.get_threshold())
+    gc.set_threshold(700, 0, 0)
```

Comment:
See https://docs.python.org/2/library/gc.html for details - this should affect how often gc runs, but it will prevent the things that stick around in gen 1 and 2 from staying there.